Calculator/Program.cs

```diff
@@ -0,0 +1,238 @@
+        static bool endApp;
+        static double firstNumber;
+        static double secondNumber;
+        
+        static Program()
+        {
+            calculator = new Calculator();
+            endApp = false;
+            firstNumber = double.NaN;
+            secondNumber = double.NaN;
+        }
+        
+        static void Main(string[] args)
+        {
+            MainMenu();
+
+            calculator.Finish();
+            return;
+        }
+
+        static void Calculate()
+        {
+            while (true)
+            {
+                // Declare variables and set to empty.
+                string numInput1 = "";
+                string numInput2 = "";
+                double cleanNum1 = 0;
+                double cleanNum2 = 0;
+                double result = 0;
+
+                // Ask the user to type the first number.
+                if (double.IsNaN(firstNumber))
+                {
+                    Console.Write("Type first number, and then press Enter: ");
+                    numInput1 = Console.ReadLine();
+
+
+                    cleanNum1 = 0;
+                    while (!double.TryParse(numInput1, out cleanNum1))
+                    {
+                        Console.Write("This is not valid input. Please enter an integer value: ");
+                        numInput1 = Console.ReadLine();
+                    }
+                }
+                else
+                {
+                    cleanNum1 = firstNumber;
+                }
+
+                // Ask the user to type the second number.
+                if (double.IsNaN(secondNumber))
+                {
+                    Console.Write("Type second number, and then press Enter: ");
+                    numInput2 = Console.ReadLine();
+
+                    cleanNum2 = 0;
+                    while (!double.TryParse(numInput2, out cleanNum2))
+                    {
+                        Console.Write("This is not valid input. Please enter an integer value: ");
+                        numInput2 = Console.ReadLine();
+                    }
+                } 
+                else
+                {
+                    cleanNum2 = secondNumber;
+                }
+
+                // Ask the user to choose an operator.
+                Console.WriteLine("Choose an operator from the following list:");
+                Console.WriteLine("Please note, only the first number will be used for square root,");
+                Console.WriteLine("10x, sine, cosine, tangent and cotangent.");
+                Console.WriteLine("\ta - Add");
+                Console.WriteLine("\ts - Subtract");
+                Console.WriteLine("\tm - Multiply");
+                Console.WriteLine("\td - Divide");
+                Console.WriteLine("\tp - Power");
+                Console.WriteLine("\tr - Square Root");
+                Console.WriteLine("\tx - 10x");
+                Console.WriteLine("\tsin - Sine");
```

Comment:
For these trigonometric functions, it doesn't seem to print the right answers when I compare it with the results from a physical calculator :sweat_smile:. Trigonometric functions only accept radians as input. do you expect the user to provide radians or degrees? If radians then its fine :blush: but if it's degrees then you will have to convert the user's input to radians.  It would be nice to specify what you expect from the user so as to enable accurate calculations. :man_technologist: